### ISSUE TITLE
bug: Slot Attribute is lost

### ISSUE THREAD

**Issue by dutscher** [ionic-team/stencil#5215]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/stencil/blob/main/CONTRIBUTING.md).
- [X] I agree to follow the [Code of Conduct](https://github.com/ionic-team/stencil/blob/main/CODE_OF_CONDUCT.md).
- [X] I have searched for [existing issues](https://github.com/ionic-team/stencil/issues) that already report this problem, without success.

### Stencil Version

4.8.2

### Current Behavior

if i set in the stencil config `experimentalSlotFixes: true` the <div slot="content" /> is relocated correctly but its loses the attribute [slot].

```html
<my-comp>
<div slot="content">Hello</div>
</my-comp>
```

-> will render into:

```html
<my-comp>
<div>Hello</div>
</my-comp>
```

### Expected Behavior

```html
<my-comp>
<div slot="content">Hello</div>
</my-comp>
```

-> will render into:

```html
<my-comp>
<div slot="content">Hello</div>
</my-comp>
```

### System Info

```shell
Node 18, Windows 11, Chrome 120
```

### Steps to Reproduce

Create a snippet with slot attribute. After first render inspect in browser the rendered component and the slot attribute is lost.
Without the experiemental flag (the default) the slot attribute is every time there.

### Code Reproduction URL

https://stackblitz.com/edit/node-uahzga?file=src%2Findex.html,src%2Fcomponents%2Fmy-comp.tsx

### Additional Information

in the 4.8.2 commit i see only a default slot unit test. maybe a named slot unit test will show this issue aswell :)
https://github.com/ionic-team/stencil/commit/4303d6af1bbcd995e3e02891b5e50768e8eeaffd

cheers

**Comment by tanner-reits** [comment]
A fix for this has been included in today's [Stencil v4.10 release](https://github.com/ionic-team/stencil/releases/tag/v4.10.0)!

**Comment by dutscher** [comment]
i can confirm! it works https://stackblitz.com/edit/node-uahzga?file=src%2Findex.html

thanks for this fix!
